packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -60,30 +66,47 @@ export const MenuItem = ({ children, onClick, to, isFocused, ...props }) => {
     }
   };
 
-  return to ? (
-    <OptionLink to={to} {...menuItemProps}>
-      <Box padding={2}>
-        <Typography>{children}</Typography>
-      </Box>
-    </OptionLink>
-  ) : (
-    <OptionButton onKeyDown={handleKeyDown} onMouseDown={onClick} type="button" {...menuItemProps}>
-      <Box padding={2}>
-        <Typography>{children}</Typography>
-      </Box>
-    </OptionButton>
-  );
+  if (to)
+    return (
+      <OptionLink to={to} {...menuItemProps}>
+        <Box padding={2}>
+          <Typography>{children}</Typography>
+        </Box>
+      </OptionLink>
+    );
+  else if (href && isExternal)
```

Comment:
after checking `v1` of `Link` component doesn't hold any `isExternal` logic 👀
```js
  else if (href)
    return (
      <OptionExternalLink {...menuItemProps}>
        <Box padding={2}>
          <ExternalLink href={href}>
            <Typography textColor="danger200">{children}</Typography>
          </ExternalLink>
        </Box>
      </OptionExternalLink>
    );
```

Reply:
Ya, I have checked recently. It is written in storybook explanation but not implemented in code. I will add the implementation.